light-system-programs/tests/merkle_tree_tests.ts

```diff
@@ -678,7 +679,7 @@ describe("Merkle Tree Tests", () => {
       TRANSACTION_MERKLE_TREE_KEY,
     );
 
-    let poseidon = await circomlibjs.buildPoseidonOpt();
+    await circomlibjs.buildPoseidonOpt();
```

Comment:
If we don't use poseidon please delete it

Reply:
removed ✅

Reply:
it's back? xD

Reply:
I don't see it!